runtime/pkg/rilltime/rilltime.go

```diff
@@ -431,3 +672,22 @@ func getMinGrain(d duration.StandardDuration) string {
 	}
 	return ""
 }
+
+// truncateWithCorrection truncates time by a grain but corrects for https://en.wikipedia.org/wiki/ISO_week_date#First_week
+// TODO: will adding this directly to timeutil.TruncateTime break anything?
```

Comment:
Should this be addressed now?

Reply:
If we move the logic to TruncateTime alerts/reports might be affected. Mainly when duration package uses it it should perhaps not have this behaviour.